src/main/java/mate/jdbc/exception/DataProcessingException.java

```diff
@@ -0,0 +1,7 @@
+package mate.jdbc.exception;
+
+public class DataProcessingException extends RuntimeException {
+    public DataProcessingException(String message, Throwable ex) {
+        System.out.println(message + ", " + ex);
```

Comment:
```suggestion
        super(message, ex);
```

Why `System.out.println` ?